libretroshare/src/util/rsstring.h

```diff
@@ -30,10 +30,10 @@ namespace librs {
 
 		bool ConvertUtf8ToUtf16(const std::string& source, std::wstring& dest);
 		bool ConvertUtf16ToUtf8(const std::wstring& source, std::string& dest);
-
+#if 0
 		bool is_alphanumeric(char c) ;
```

Comment:
has these two been moved somewhere else?

Reply:
They are not used anymore so I kept them under `#ifdef 0` if we need them in the future.